frontend/src/components/forms/CreateLog.tsx

```diff
@@ -273,9 +325,9 @@ const CreateLog = () => {
                   <FormControl isRequired>
                     <FormLabel>Employee</FormLabel>
                     <Select
-                      options={EMPLOYEES}
+                      options={employeeOptions.length > 0 ? employeeOptions : []}
```

Comment:
Don't think you need this ternary since it's an empty array by default.

Same applies for cases below